### ISSUE TITLE
Accessibility: Label for selected filter is not visible 

### ISSUE THREAD

**Issue by spacefront** [contao/contao#6021]
### Description

![image](https://user-images.githubusercontent.com/6522818/236666224-ab7d6841-21ce-43f1-91c5-8958ea68dec5.png)

Label sollten gesetzt werden damit die Barrierefreiheit gegeben ist auch die Übersichtlichkeit für den normalen Redeakteure wird erhöht.

**Comment by zonky2** [comment]
As discussed on Contao Camp 2023 / https://github.com/contao-community-alliance/contao-camps/blob/main/2023/camp-2023.md#samstag-06052023

**Comment by gmpf** [comment]
You can assign this to me.

**Comment by aschempp** [comment]
Any idea what this could look like?

**Comment by spacefront** [comment]
gmpf can show it - he did it during the camp

**Comment by aschempp** [comment]
closing in favor of https://github.com/contao/contao/pull/6146
